contour/views.py

```diff
@@ -1104,3 +1109,95 @@ def get_info_from_reader(reader):
         var_symbols.append(reader[i])
         i += 1
     return var_names, var_symbols
+
+
+def sort_plotted_figures(probabilistic_model):
+    """
+    Sorts the fit images of a probabilistic model to generate clear overview
```

Comment:
"Sorts the images showing the fits to generate a structured overview in the template" ?